test/compress/class-properties.js

```diff
@@ -246,3 +246,32 @@ mangle_class_properties_keep_quoted: {
         }
     }
 }
+
+private_class_properties: {
+    no_mozilla_ast = true;
+    node_version = ">=12";
+    options = {
+        ecma: 2015
+    }
+    input: {
+        class Foo {
+            #bar = "FooBar"
```

Comment:
Could add `static` privates to this (or another) fixture as well. Trying it out locally they behave as expected!

Reply:
I can try to work on adding a set of more checks to that PR if that is ok with @L2jLiga and @fabiosantoscode. We're pretty interested in a way to land that PR or a similar one.